addons/frag/functions/fnc_addBlacklist.sqf

```diff
@@ -0,0 +1,29 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Jaynus, NouberNou
+ * Adds a round to the blacklist (will be ignored).
+ *
+ * Arguments:
+ * 0: Projectile <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_projectile] call ace_frag_fnc_addBlackList
+ *
+ * Public: No
+ */
+
+params ["_projectile"];
+TRACE_2("addBlackList",_projectile,typeOf projectile);
+
+_projectile setVariable [QGVAR(blacklisted)];
```

Comment:
Value is missing

Reply:
How do I keep doing this?
Fixed here: https://github.com/acemod/ACE3/pull/10157/commits/7b282ec96cfb1130283858f56762c4f2a2e45928